.nvmrc

```diff
@@ -1 +1 @@
-14.16.1
+16.16.0
```

Comment:
❤️

Reply:
I couldn't actually install deps using 14 due to... reasons